code-build.js

```diff
@@ -110,6 +110,8 @@ async function waitForBuildEndTime(sdk, { id, logs }, nextToken) {
 
 function githubInputs() {
   const projectName = core.getInput("project-name", { required: true });
+  const disableSourceOverride =
```

Comment:
nit - For consistency, could you either update this new line to use core.getInput("disable-source-override", {required: false}) === "true", or remove all other { required: false }s from githubInputs()?